command/server.go

```diff
@@ -371,6 +371,19 @@ func (c *ServerCommand) Run(args []string) int {
 	// Initialize the listeners
 	lns := make([]net.Listener, 0, len(config.Listeners))
 	for i, lnConfig := range config.Listeners {
+		if lnConfig.Type == "atlas" {
+			cluster, err := core.Cluster()
```

Comment:
Oh, geez. I didn't realize that the cluster info was behind the barrier. Does that mean that `/sys/health` doesn't return `cluster_id` when the instance is sealed?